<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>833</width>
    <height>473</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="editShareTableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>761</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="editShareDateEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>110</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="editSharePortionComboBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>310</y>
     <width>120</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>100</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Jakopäivä</string>
   </property>
  </widget>
  <widget class="QPushButton" name="editShareSavePushButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>350</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Tallenna</string>
   </property>
  </widget>
  <widget class="QPushButton" name="editShareCancelPushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Peruuta</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="editSharePopulatePushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Täytä</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>290</y>
     <width>120</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Ruhonosa</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>120</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Jaot</string>
   </property>
  </widget>
  <widget class="QLabel" name="editShareChosenLabel">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>264</y>
     <width>291</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Ei valittua jakoa</string>
   </property>
  </widget>
  <widget class="QComboBox" name="editShareMemberComboBox">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>310</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="editSharePartyComboBox">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>310</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="shotByLabel_13">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>290</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Jäsen</string>
   </property>
  </widget>
  <widget class="QLabel" name="shotByLabel_14">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>290</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Seurue</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
